scripts/clean_unused_imports.py

```diff
@@ -2,10 +2,15 @@
 import os
 import re
 
-IMPORT_RE = re.compile(r'^\s*import\s+[\w\.]+\.(\w+)\s*;\s*(?://.*)?$')
-
 LANGUAGE_REGEX_MAP = {
-    "java": "*.java"
+    "java": {
+        "pattern": "*.java",
+        "regex": re.compile(r'^\s*import\s+[\w\.]+\.(\w+)\s*;\s*(?://.*)?$')
+    },
+    "php": {
+        "pattern": "*.php",
+        "regex": re.compile(r'^\s*use\s+[\w\\]+\\(\w+)\s*\w*\s*(\w*);\s*(?://.*)?$')
```

Comment:
Should consider using existing tooling like https://github.com/PHP-CS-Fixer/PHP-CS-Fixer